share/retriever/retriever.go

```diff
@@ -19,11 +22,11 @@ import (
 	"go.opentelemetry.io/otel/codes"
 	"go.opentelemetry.io/otel/trace"
 
-	"github.com/celestiaorg/celestia-node/ipld/plugin"
 	"github.com/celestiaorg/nmt"
 	"github.com/celestiaorg/rsmt2d"
 )
 
+// TODO: fix tracer and loggegr everywhere
```

Comment:
typo in logger, but what does this mean?

Reply:
gone, was left from WIP